tests/SideBySide.New/StoredProcedureTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Data;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using Xunit;
+
+namespace SideBySide
+{
+	public class StoredProcedureTests : IClassFixture<DatabaseFixture>, IDisposable
+	{
+		public StoredProcedureTests(DatabaseFixture database)
+		{
+			m_database = database;
+			m_database.Connection.Open();
+		}
+
+		public void Dispose()
+		{
+			m_database.Connection.Close();
+		}
+
+		[Fact]
+		public async Task StoredProcedureBasic()
+		{
+			using (var cmd = m_database.Connection.CreateCommand())
+			{
+				cmd.CommandText = @"DROP PROCEDURE IF EXISTS simple;
```

Comment:
FWIW, the exception is being thrown from line 45, i.e., `ExecuteNonQueryAsync` on the stored procedure.